### DISCUSSION TITLE
Senec playing around with numbers again?

### DISCUSSION THREAD

**Discussion by scottgoo** [marq24/ha-senec-v3#91] [General]
I got strange numbers in energy dashboard yesterday (28th of june).

![image](https://github.com/marq24/ha-senec-v3/assets/38814097/8ac9d8b6-73ff-4810-a0f6-4cf3ae656981)
No solar production but it should has been more than 50 kWh over the day. 

The day before everything was fine. Today things are slowly going back to normal. Solar production seems to be correct since 9 am. But there should have been some solar power since 5 am.
![image](https://github.com/marq24/ha-senec-v3/assets/38814097/bc644df4-c7df-431b-80d8-d044c1cddcf5)
Anyone else with this issues? Is senec messing around with data again?

**Comment by ETH1166** [comment]
There is a discussion listed under issue #90, where systems show similar behaviour: https://github.com/marq24/ha-senec-v3/issues/90#issuecomment-2197047373
